tests/integration/profiles_management/test_create_or_update_profiles.py

```diff
@@ -49,3 +56,68 @@ async def test_remove_access_to_stale_profiles(
 
     log.info("Removing test Profile and resources in it.")
     profiles.remove_profile(profile, lightkube_client)
+
+
+@pytest.mark.asyncio
+async def test_new_profiles_created(lightkube_client: Client):
+    pmr = classes.ProfilesManagementRepresentation()
+
+    users = ["noha", "orfeas"]
+    expected_quota = classes.ResourceQuotaSpecModel.model_validate({"hard": {"cpu": "1"}})
+    for user in users:
+        pmr.add_profile(
+            classes.Profile(
+                name=user,
+                owner=classes.Owner(name=user, kind=classes.UserKind.USER),
+                resources=expected_quota,
+            )
+        )
+
+    create_or_update_profiles(lightkube_client, pmr)
+
+    log.info("Will check if Profiles were created as expected")
+    for user in users:
+        created_profile = profiles.get_profile(lightkube_client, user)
+        created_profile_quota = classes.ResourceQuotaSpecModel.model_validate(
+            created_profile["spec"]["resourceQuotaSpec"]
+        )
+        assert created_profile_quota == expected_quota
+        profiles.remove_profile(created_profile, lightkube_client)
+
+
+@pytest.mark.asyncio
+async def test_update_resource_quota(lightkube_client: Client):
+    profile_path = TESTS_YAMLS_PATH + "/profile.yaml"
+    log.info("Loading test yamls from: %s" % profile_path)
```

Comment:
suggestion: Not sure if you're using `%` for a specific reason, let's just use commas
```suggestion
    log.info("Loading test YAMLs from: %s.",  profile_path)
```